core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -147,41 +147,17 @@ protected function compile()
 			}
 
 			$arrResult = null;
```

Comment:
This line can be removed I think because it gets set both in the `try` and in the `catch` block.

Reply:
Changed in 4116d124af8d2e63077846cdad61a147e3c87045.